packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1117,6 +1117,11 @@ abstract class SliderTrackShape {
     bool isDiscrete,
     required TextDirection textDirection,
   });
+
+  /// Whether the track shape is rounded.
+  ///
+  /// This is used to determine the correct position of the thumb in relation to the track.
+  bool get isRounded;
```

Comment:
Is is possible to give a default value here? This will break anyone that `extends` SliderTrackShape.

Reply:
Thank you!